### DISCUSSION TITLE
Multiple tabs not possible in Version 24.1.4.

### DISCUSSION THREAD

**Discussion by rnuxip** [dbeaver/dbeaver#35166] [General]
After updating to Version 24.1.4 its not longer possible to have multiple tabs. Even if a tab is pinned and a new query executed, the results are always shown in the same tab.
Changing  Sql Editor -> Layout -> ;show results in a single/multiple Tabs' does not work.

This Versions seems to be buggy!

**Comment by E1izabeth** [comment]
We fixed this bug.
You can try Early Access version with fix (https://dbeaver.io/files/ea/) or wait for the next release
